package.json

```diff
@@ -92,17 +96,12 @@
     "react-native-listener": "1.0.1",
     "react-redux": "4.4.6",
     "react-webcam-onfido": "0.0.19",
-    "redux": "3.5.2",
-    "supports-webp": "1.0.3",
-    "visibilityjs": "1.2.4",
-    "wpt": "https://github.com/onfido/js-client-tracker/tarball/master",
-    "babel-runtime": "6.23.0",
-    "deep-equal": "1.0.1",
     "reconnectingwebsocket": "1.0.0",
-    "eventemitter2": "2.1.3",
-    "query-string": "4.1.0",
     "redux": "3.4.0",
```

Comment:
Redux was defined twice in the dependencies. You removed 3.5.2. I'm not sure which version we were actually using, but we should make sure we are still using this version.